tests/functional/test_rose_stem.py

```diff
@@ -251,234 +252,83 @@ def _inner_fn(rose_stem_opts, verbosity=verbosity):
     yield _inner_fn
 
 
-@pytest.fixture(scope='class')
```

Comment:
Class-scoped fixture wasn't needed as it was only used once (it's a bit too specific to the test to be reusable).

Reply:
Much of the class scoped stuff was done in the name of doing as little re-writing as possible when moving the tests from Stuarts work.